composer.json

```diff
@@ -22,7 +22,7 @@
         }
     ],
     "require": {
-        "php": ">=8.1",
+        "php": ">=8.0",
```

Comment:
I don't think the project will switch to 8.0. You may use 4.x in that case.

Or is there any issue using 4.x ?

Reply:
i reduced the requirement because the project runs perfectly fine on php 8.0 and does not need to be restricted to `>8.1`

Reply:
The discussion around this happened elsewhere and it was determined that Aura 5.x was 8.1+ only. Even though the codebase works fine on 8.0 now, maintainers wanted to give themselves the option to add 8.1+ syntax later without breaking compat with anyone who has 5.x already installed. Which is why I built out 4.x as a bridge up to 8.1.

Reply:
I agree with @iansltx . If @r4ndsen can revert the changes for 8.0 , I will merge ASAP and make a release. Else it may take some time than expected.

Reply:
i adjusted the `5.x` branch

Reply:
Cool. Thank you.